src/Test.Utilities/CSharpProjectState.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Testing;
+
+namespace Microsoft.NetCore.Analyzers.Security.UnitTests
+{
+    /// <summary>
+    /// This class is a copy-paste from
+    /// https://github.com/dotnet/roslyn-sdk/blob/master/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.CSharp.Analyzer.Testing/CSharpProjectState.cs
+    /// </summary>
+    public class CSharpProjectState : ProjectState
```

Comment:
The `ProjectState` class is not available. It seems that the dll does contain the class and that the class is `public` so I am not really sure what's missing. Could it be a different analyzer version compared to master?

Reply:
No longer valid issue.